src/collection-preferences/content-display/content-display-option.tsx

```diff
@@ -0,0 +1,53 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import styles from '../styles.css.js';
+import DragHandle from '../../internal/drag-handle';
+import InternalToggle from '../../toggle/internal';
+import React, { ForwardedRef, forwardRef } from 'react';
+import { SyntheticListenerMap } from '@dnd-kit/core/dist/hooks/utilities';
+import { OptionWithVisibility } from './utils';
+import { useUniqueId } from '../../internal/hooks/use-unique-id';
+
+const componentPrefix = 'content-display-option';
+export const getClassName = (suffix?: string) => styles[[componentPrefix, suffix].filter(Boolean).join('-')];
+
+export interface ContentDisplayOptionProps {
```

Comment:
Following the "presentational component" pattern as documented in the [dnd-kit documentation](https://docs.dndkit.com/api-documentation/draggable/drag-overlay#presentational-components), this component is used in two ways:
- For the options displayed in the list, wrapped in DraggableOption
- While dragging, to render the active (i.e, dragged) item wrapped in the DragOverlay